packages/sample-apps/windows/NuGet.Config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="repositoryPath" value="packages" />
+  </config>
+  <packageSources>
+    <clear />
+    <add key="react-native" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
```

Comment:
This might trigger the single feed policy on publish. Don't know if we blanket allow or have to whitelist each individual one.
@jonthysell ?

Reply:
This was copied from the NuGet.config used by other apps, from the template.